### ISSUE TITLE
Program Can't Start - api-ms-win-core-path...

### ISSUE THREAD

**Issue by Gmanicus** [Gmanicus/MC-Outpost#1]
On other systems, the application appears to fail to start due to missing dll files while outside of the development environment.

**Comment by Gmanicus** [comment]
Added `--paths C:\Windows\System32\downlevel` to build script to include these missing dll files.

Waiting for tests to see if issue is resolved.

**Comment by Gmanicus** [comment]
Testing has revealed that the executable can only be built for the OS that the development environment is in (I have a Windows 10 machine, making the executable incompatible with Windows 7 machines). I'll see if I can outsource a Windows 7 building environment.

**Comment by Gmanicus** [comment]
Built for Windows 7, but ran into multitudes of compatibility issues.

- The most up-to-date version of Python no longer supports Windows 7
- The requirements needed to be customized, downgrading several included libraries, which likely led to further errors
- Several necessary libraries were not included in the requirements for some reason
- GenericInputDialog was no longer included in the build for some reason
- GenericInputDialog had issues with `from __future__ import annotations`. I tried removing this, as apparently this line is new and potentially only necessary past Python 3.7 (I was attempting to build with 3.6), but ran into further errors.

It was at this point that I decided compatibility with Windows 7 was not worth it. While this may not seem like much, this was a time-consuming process rebuilding, testing, rebuilding, and testing. Compatibility with Windows 7 may not even be possible.
